src/file-selector.ts

```diff
@@ -110,7 +110,7 @@ function toFilePromises(item: DataTransferItem) {
         return fromDirEntry(entry) as any;
     }
 
-    return fromDataTransferItem(item);
```

Comment:
This would be a breaking change.

Reply:
@rolandjitsu I changed this to still call fromDataTransferItem while still getting the functionality I need.